client/apps/edit/components/layout/index.coffee

```diff
@@ -156,18 +156,26 @@ module.exports = class EditLayout extends Backbone.View
 
   checkSeo: =>
     imageCount = 0
+    textIndex = 0
     @fullText = []
-    @fullText.push(" <p> " + $(@article.get("lead_paragraph")).text() + " </p> ")
-    for section in $(@article.get("sections"))
-      if section.type is "text"
+    if ($(@article.get('lead_paragraph')).text() != '')
```

Comment:
actually what's even better is `if @article.get('lead_paragraph').text()` because an empty text will be a falsey value